sdk/communication/AzureCommunication/Tests/CommunicationIdentifierSerializerTests.swift

```diff
@@ -150,69 +172,23 @@ class CommunicationIdentifierSerializerTests: XCTestCase {
     }
 
     func test_SerializeMicrosoftTeamsUser_NotAnonymous_ExpectedIdString() throws {
-        try serializeMicrosoftTeamsUser(isAnonymous: false, expectedId: "some id")
+        try serializeMicrosoftTeamsUser(isAnonymous: false, expectedId: testRawId)
     }
 
     func test_SerializeMicrosoftTeamsUser_Anonymous_ExpectedIdString() throws {
-        try serializeMicrosoftTeamsUser(isAnonymous: true, expectedId: "some id")
+        try serializeMicrosoftTeamsUser(isAnonymous: true, expectedId: testRawId)
     }
 
     func serializeMicrosoftTeamsUser(isAnonymous: Bool, expectedId: String?) throws {
-        let model = try CommunicationIdentifierSerializer
-            .serialize(identifier: MicrosoftTeamsUserIdentifier(
-                userId: "user id",
-                isAnonymous: isAnonymous,
-                identifier: expectedId
-            ))
-
-        XCTAssertEqual(model.kind, .microsoftTeamsUser)
-        XCTAssertEqual(model.microsoftTeamsUserId, "user id")
-        XCTAssertEqual(model.isAnonymous, isAnonymous)
-        XCTAssertEqual(model.id, expectedId)
-    }
-
-    func test_DeserializeMissingProperty() throws {
-        let modelsWithMissingMandatoryProperty: [CommunicationIdentifierModel] = [
-            CommunicationIdentifierModel(kind: .unknown), // Missing Id
```

Comment:
each of these model is a different test cases. I see the code is removed. So are we not testing these anymore?

Reply:
In theory it is impossible to have a missing property in the model the way it is set up I think. I will double check